stdpopsim/citations.py

```diff
@@ -24,4 +24,7 @@ class Citation(object):
     year = attr.ib(type=int, kw_only=True)
 
     def __str__(self):
-        return f"{self.author}, {self.year}: {self.doi}"
+        if self.author is not None:
+            return f"{self.author}, {self.year}: {self.doi}"
+        else:
```

Comment:
Is there a valid reason for not having an author and year? If not, I think this will just mask mistakes. Better for something to be obviously wrong so that we can fix it I think.

Reply:
I didn't look to see how widespread it is, but at least the `HapmapII_GRCh37` genetic map has a url-only citation, with the url stuffed into the doi field.

Reply:
Ok, I removed this change now.

Reply:
Great --- I got rid of the URL citation in HapmapII_GRCh37